### PR TITLE
Added Euler_Totient_Function.c and README.md file

### LINKED ISSUE #1098: Euler Totient Function in C

### PR DESCRIPTION
**Pull request by Vaibhavraj10**
Fixes #1098 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:

- Added Euler Totient Function in C language 
- Added a README.md file.

#### Languages Used:
- C

#### Files Added:
- Euler_Totient_Function.c
- README.md

### CODE REVIEW

File: Euler_Totient_Function/Euler_Totient_Function.c [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    if(a == 0)
+        return 0;
+    return gcd(b % a, a);
+}
+
+// Euler totient Function by using Brute force method, checking GCD of every number with n, if equals 1.
+ll EulerTotient_Brute(ll num)
+{
+    ll answer = 1;
+    for(ll i = 2; i<num; i++)
```
Comment (somya-kapoor) [inline Euler_Totient_Function/Euler_Totient_Function.c]: Please add space around <.

File: Euler_Totient_Function/Euler_Totient_Function.c [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+// Main method
+int main()
+{
+    ll num;
+    printf("Enter the number: \n");
+    scanf("%lld", &num);
+    printf("Euler totient function value is: %lld", EulerTotient_ProductForm(num));
+    return 0;
+
```
Comment (somya-kapoor) [inline Euler_Totient_Function/Euler_Totient_Function.c]: Remove line 52

File: Euler_Totient_Function/Euler_Totient_Function.c [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+}
```
Comment (somya-kapoor) [inline Euler_Totient_Function/Euler_Totient_Function.c]: Add a line between the end of code and input

File: Euler_Totient_Function/Euler_Totient_Function.c (hunk 1 above)
Comment (Vaibhavraj10) [reply Euler_Totient_Function/Euler_Totient_Function.c]: Added.

File: Euler_Totient_Function/Euler_Totient_Function.c (hunk 2 above)
Comment (Vaibhavraj10) [reply Euler_Totient_Function/Euler_Totient_Function.c]: Removed.

File: Euler_Totient_Function/Euler_Totient_Function.c (hunk 3 above)
Comment (Vaibhavraj10) [reply Euler_Totient_Function/Euler_Totient_Function.c]: Added.

**Pull request merged**